FILE packages/website/public/og-image.png
RANGE {"path":"packages/website/public/og-image.png","old_lines":null,"new_lines":null,"diff_block_lines":[1,3],"hunk_header":null,"substantive_changed_lines":0}
diff --git a/packages/website/public/og-image.png b/packages/website/public/og-image.png
index f9b8c171d..acd25d018 100644
Binary files a/packages/website/public/og-image.png and b/packages/website/public/og-image.png differ
